front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/pipeline/Pipeline.groovy

```diff
@@ -22,9 +22,6 @@ import com.netflix.spinnaker.front50.model.Timestamped
 
 class Pipeline extends HashMap<String, Object> implements Timestamped {
   @JsonIgnore
-  Long lastModified
```

Comment:
I don't believe having this actually go across the wire will cause grief but I'll give it a whirl in prestaging.